packages/transformers/webextension/src/WebExtensionTransformer.js

```diff
@@ -75,10 +77,12 @@ async function collectDependencies(
       });
     }
     for (const locale of await fs.readdir(locales)) {
-      asset.addURLDependency(`_locales/${locale}/messages.json`, {
-        needsStableName: true,
-        pipeline: 'raw',
-      });
+      if (await fs.exists(path.join(locales, locale))) {
```

Comment:
Is this check needed? You just read the directory, it's unlikely the file got deleted in the meantime.

Reply:
Now that you mention it that does seem weird but I think it fixed an issue where files like `.DS_Store` were accessed? I'll check again.

Reply:
Oh right I was going to check if `messages.json` existed but I accidentally forgot to write out the full path. Fixing now.